azure/packages/external-controller/src/app.ts

```diff
@@ -104,12 +105,16 @@ async function initializeNewContainer(
 }
 
 async function start(): Promise<void> {
```

Comment:
FOR REVIEWERS:

This is an example of what the user experience would look like for initializing the devtools as a part of their app flow. Interested in thoughts 🙂